dsl/docker-multiarch/images/openjdk.groovy

```diff
@@ -0,0 +1,44 @@
+import vars.multiarch
+
+for (arch in multiarch.allArches()) {
+	meta = multiarch.meta(getClass(), arch)
+
+	freeStyleJob(meta.name) {
+		description(meta.description)
+		logRotator { daysToKeep(30) }
+		label(meta.label)
+		scm {
+			git {
+				remote { url('https://github.com/docker-library/openjdk.git') }
+				branches('*/master')
+				clean()
+			}
+		}
+		triggers {
+			upstream("docker-${arch}-debian, docker-${arch}-buildpack", 'UNSTABLE')
+			scm('H H/6 * * *')
+		}
+		wrappers { colorizeOutput() }
+		steps {
+			shell(multiarch.templateArgs(meta, ['dpkgArch']) + '''
+case "$dpkgArch" in
+	ppc64el)
+		sed -i 's!amd64!ppc64el!g' */Dockerfile
```

Comment:
Couldn't this bit be generalized to something like `sed -i "s!amd64!$dpkgArch!g" */Dockerfile` (and thus moved out of the case)?  Wouldn't it need to be in order to support arches that aren't `ppc64le`?  (We support `i386` which should build fine on an `amd64` box, so you could add a 32bit node to your Jenkins to test these on too, which might help in this regard.)